code/game/objects/items/weapons/cigs_lighters.dm

```diff
@@ -328,6 +328,44 @@ CIGARETTE PACKETS ARE IN FANCY.DM
 	type_butt = /obj/item/trash/cigbutt/khi
 	preloaded_reagents = list("mindbreaker" = 5, "serotrotium" = 5, "impedrezene" = 5, "space_drugs" = 5)
 
+/obj/item/clothing/mask/smokable/cigarette/comred
+	name = "\improper ComRed light cigarette"
+	type_butt = /obj/item/trash/cigbutt/comred
+	icon_state = "comredoff"
+	icon_on = "comredon"
+	icon_off = "comredoff"
+	preloaded_reagents = list("nicotineplus" = 5, "vodka" = 10)
```

Comment:
A communist cig with fine tobacco from the cigars, AND vodka? I *need* to try this one!